devicetypes/smartthings/zwave-switch-secure.src/zwave-switch-secure.groovy

```diff
@@ -119,7 +117,7 @@ def refresh() {
 }
 
 private command(physicalgraph.zwave.Command cmd) {
-	if (state.sec != 0) {
+	if (zwaveInfo && zwaveInfo.zw?.endsWith("s")) {
```

Comment:
We should probably just use:

`zwaveInfo?.zw?.contains("s")`

I am not 100% sure that we are guaranteed that "s" is on the end. (In fact, we should probably make sure all instances of the code like this is made uniform.) cc @kianooshST  @mckeed